<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSectionWindow</class>
 <widget class="QDialog" name="AddSectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Price $</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="avaiBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Available</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
